doc/tutorial/tutorial.mdk

```diff
@@ -1175,6 +1212,11 @@ let rec append_mem l1 l2 a = match l1 with
 ~~
 ~
 
+~KK 
+Implicit type parameters were not yet introduced, neither was the "hasEq"
+function or what it means. The implicity type parameters are not visible in the
+code snippet above, but are present in the code that is loaded into the editor.
+~

```

Comment:
This also confused me. Coming from Coq I inferred (no pun intended) what #t and `hasEq` were, but I first had to check the exercise code to figure out how to add this assumption on the type.

Also, later on the alias `type eqtype = t:Type{hasEq t}` from prims is used consistently, again without explanation.